Makefile

```diff
@@ -54,3 +55,17 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+builder-rpm:
+	cd $(WD)/packages && docker build -t calico-mesos-builder .
```

Comment:
We can assume that Makefile is being executed in the working directory, so you should be able to omit the `$(WD)` from these commands